ext/pdo/php_pdo.h

```diff
@@ -18,6 +18,7 @@
 #define PHP_PDO_H
 
 #include "zend.h"
+#include "pdo_arginfo.h"
```

Comment:
Was this include put in the right place?

Reply:
The include should be in the `.c` file.